README.md

```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
 ## Text Classification 
 
 
+Initialize a HappyTextClassification() object to perform text classification. 
+
+This model assigns a label to a given text string. For example, you can train a model to 
+detect if an email is spam based on its text. 
+
+
+Initialization Arguments: 
+    1. model_type (string): either "ALBERT", "BERT" or "DISTILBERT." The default is "DISTILBERT"
+    2. model_name(string): below is a URL that contains potential models. The default is "distilbert-base-uncased"
+       [MODELS](https://huggingface.co/models?filter=text-classification)
+    3. num_labels(int): The number of text categories. The default is 2
+    
+WARNING: If you try to load a pretrained model that has a different number of categories 
+than num_labels, then you will get an error 
+
+"albert-base-v2", "bert-base-uncased" and "distilbert-base-uncased" do not have a predefined 
+number of labels, so if you use these models you can set num_labels freely 
+
+### Initialization  
+
+"HappyTextClassification("ALBERT", "textattack/albert-base-v2-SST-2")"  has many useful applications. 
+It's able to detect the sentiment of text.  
```

Comment:
This statement implies that this ALBERT model is the only one that has these abilities. Seems confusing

Reply:
Fixed in d3b4534